src/cache.rs

```diff
@@ -506,8 +534,7 @@ where
                 let (parts, body) = response.into_parts();
                 match self.storage.store(key.clone(), parts, body, policy).await {
                     Ok(mut response) => {
-                        // Response was stored, return the response with the storage key
-                        response.set_cache_status(CacheLookupStatus::Hit, CacheStatus::Miss);
+                        response.set_cache_status(CacheLookupStatus::Hit, CacheStatus::Miss, None);
```

Comment:
Based on the description of the PR, I'm a bit surprised this doesn't return a digest. Why is that here?